packages/core/src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,38 @@
+import clsx from "clsx";
+import { HvBaseProps } from "index";
```

Comment:
We need to import from `"../../types"` for the props to appear in the storybook.